integrated_channels/blackboard/client.py

```diff
@@ -147,7 +147,9 @@ def delete_content_metadata(self, serialized_data):
         BlackboardAPIClient._validate_channel_metadata(channel_metadata_item)
         external_id = channel_metadata_item.get('externalId')
         course_id = self._resolve_blackboard_course_id(external_id)
-        BlackboardAPIClient._validate_course_id(course_id, external_id)
+
```

Comment:
I suggested adding a unit test to test we don't fail when course id is not found from LMS side, for Bb client